pkg/controller/controller.go

```diff
@@ -63,7 +63,7 @@ func (c *SecretSyncController) SyncAll() {
 		if err != nil {
 			klog.Errorf("Secret sync failed for %s: %s", spec, err)
 		}
-		if updated {
+		if updated && c.PrintMsg {
```

Comment:
During the demo, I thought it might be helpful to have an option to run the controller in a "quiet mode", only showing error messages instead of all sync infos?

Reply:
I'm not sure if we would prefer printing an info every time the controller syncs a pair?

Reply:
This is something I would typically rely on my logging library for.  In this case I suspect you could use `klog.InitFlags(nil)` to expose klog's flags, and use something like `--v` to specify what you want to see at runtime

If you don't feel like figuring out klog maybe rename the flag to something like 'Verbose'?

Reply:
Yes! verbose is the term I'm looking for.
After reading the https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
I think I'll put the sync info on level 2, and the demo report text on level 1.
The demo will run on log level 1.